addons/product/report/product_pricelist_report.py

```diff
@@ -22,7 +22,7 @@ def get_html(self):
             self.env.context.get('pricelist_id'),
             self.env.context.get('quantities') or [1]
         )
-        return self.env.ref('product.report_pricelist_page')._render(render_values)
+        return self.env.ref('product.report_pricelist_page')._render({**render_values, 'isVisibleTitle': self.env.context.get('isVisibleTitle', False)})
```

Comment:
Move the specification of `isVisibleTitle` to `render_values` instead, why do you modify this line ?

Reply:
@Feyensv  Changes are done.
Thanks